cmd/hibernate/cluster/cmd.go

```diff
@@ -55,7 +56,7 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
-	if !confirm.Confirm("hibernate cluster %s", clusterKey) {
+	if !confirm.Yes() && !confirm.Confirm("hibernate cluster %s", clusterKey) {
```

Comment:
Please check to make sure, but I think you can use confirm.Prompt instead of confirm.Confirm when you want it to be used alongside confirm.Yes